code/style.css

```diff
@@ -2,6 +2,340 @@
+  color: whitesmoke;
+
+}
+
+.logo-mobil {
+  font-size: 25px;
+  font-weight: 600;
+  color:white;
+}
+
+.navbar {
+  display: none;
+}
+
+.navbar ul li {
+  list-style: none;
+}
+
+/* Mobil layout -- Signup form*/
+
+.form-wrapper {
+  display: flex; 
+  flex-direction: column;
+  justify-content: top;
+  height: fit-content;
+  border-radius: 15px;
+  background-color: #E2E5DE;
+  padding: 25px;
+  margin-top: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+  margin-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.form-wrapper h2 {
+  text-transform: uppercase;
+  font-size: 1rem;
+}
+
+.form-inputs {
+  display: flex;
+  flex-direction: column;
+  margin-top: 5px;
+  font-size: 1.5vh;
+  text-transform: uppercase;
+} 
+
+.form-inputs .textarea {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-inputs .textarea input {
+  padding: 3px 8px;
+  border-radius: 5px;
+  border: solid 1px #808080;
+
+}
+
+.form-inputs .textarea label {
+  padding-top: 3px
+}
+
+.form-inputs .radiobutton {
+  padding-top: 3px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.form-inputs .checkbox {
+  padding-top: 3px;
+  padding-bottom: 3px;
+}
+
+.form-inputs button {
+  border: solid 1px #808080;
+  border-radius: 5px;
+  padding: 3px 8px;
+}
+
+/*---End of form-styling */
+
+/* Mobil layout -- Accordion*/
+
+.accordion-container {
+  display: flex; 
+  flex-direction: column;
+  justify-content: top;
+  height: auto;
+  border-radius: 15px;
+  background-color: #E2E5DE;
+  padding: 20px;
+  margin-top: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+  margin-bottom: 10px;
+}
+
+.accordion-container h1 {
+  padding-bottom: 5px;
+  text-transform: uppercase;
+  font-size: 1rem; 
+}
+
+.question {
+  height: fit-content;
+  background-color: #B2BEB5;
+  border: solid 1px #808080;
+  border-radius: 10px;
+  cursor: pointer;
+  font-size: 18px;
+  padding: 3px 8px;
+}
+
+.question::after {  /*this adds an upside-down triangle to each section*/
+  content: '\25be';
+  float: right;
+  transform: scale(1); 
+}
+
+.answer {
+  display: none;
+}
+
 .active + .answer {
-  /* some styling here */
+  display: block;
+  background-color: #F5F5F5;
+  color: black;
+  padding: 3px 8px;
+  border-radius: 5px;
+  
+}
+
+/*------TABLET STYLING------*/
+
+@media (min-width: 667px) and (max-width: 1024px) {
+
+  .main-container{
+    display:grid;
+    grid-template-columns: 49% 49%;
+    grid-template-rows: 100%;
+    gap: 20px;
+    background-color: #F9F9F3;
+    margin-left: 90px;
+    margin-right: 90px;
+  }
+
+  .hero-image {
+    width: 100vw;
+    height: 100vh;
+    background-size: cover;
+    background-position: center;
+    display: grid;
+    grid-template-rows: 10vh;
+  }
+
+  .inner-hero-header { 
+    width: 100vw;
+    height: 8vh;
+    background-color: gray;
+    color: #fff;
+    opacity: .7;
+    margin-top: 0;
+  }
+    
+  .logo-place {
+    display: none;
+  }
+
+  .logo-mobil {
+    display: none;
+  }
```

Comment:
Small comment - I don't think you need to add `display: none;` to `.logo-mobil` when you already have it for it's parent div, `.logo-place`. So this could probably be deleted without changing your site.